acceptance/analyzer_test.go

```diff
@@ -395,15 +400,13 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 
 		it("drops privileges", func() {
 			h.SkipIf(t, runtime.GOOS == "windows", "Not relevant on Windows")
-			imageName := authRegistry.RepoName("some-image")
-			authConfig, err := auth.BuildEnvVar(authn.DefaultKeychain, imageName)
-			h.AssertNil(t, err)
+			imageName := fixtures.readWriteAppImage
```

Comment:
Do we even need this variable? We could use `fixtures.readWriteAppImage` on line 414...

Reply:
Fixed in the following [commit](https://github.com/buildpacks/lifecycle/pull/685/commits/7ce8bb8297e15af0aab609e3df4c462172972ef9)